content/docs/Core principles/what-is-beehaw.md

```diff
@@ -3,25 +3,25 @@ title: What is Beehaw?
+It gets even more complicated when you consider someone who is sharing an opinion they have which is actively harmful to many individuals in the world, but it’s due to their *ignorance*. We personally believe that so long as this person is not actively spreading this intolerant viewpoint and is working on themself to become a better person, that it would not be particularly productive to launch into a tirade against them. We understand, however, how someone could be quite rude in response to such intolerance and we agree that this person may desperately need to be educated appropriately. However, there is no way for that discussion to happen on this platform in a productive manner while lobbing insults at each other. We can understand why, at first brush, some might consider this tone policing. This is all to say: we disapprove of the intolerant viewpoint, and we approve of it being corrected, but we also approve of the intolerant person attempting to become a better person.
 
-The only way for a platform which is hoping to exist as an explicitly nice place online to avoid taking sides in a situation like this is to *withdraw* from the quandary entirely. This kind of nuanced political and philosophical discussion is just simply not meant for Beehaw. We’d like to think that I’m aware and learned enough to avoid ‘debating’ things like [phrenology](https://en.wikipedia.org/wiki/Phrenology), which are obviously racist, but I’m also smart enough to realize that there’s likely some ideas which we’ve internalized or been taught by a colonialist western society which are harmful to other minorities. We want to be able to learn about how everything we were taught was wrong, to be corrected, and that space can only exist when we don’t let people berate each other over ideas they project on others (whether that projection happens to be correct or not).
+The only way for a platform which is hoping to exist as an explicitly nice place online to avoid taking sides in a situation like this is to *withdraw* from the quandary entirely. This kind of nuanced political and philosophical discussion is just simply not meant for Beehaw. We’d like to think that we're aware and learned enough to avoid ‘debating’ things like [phrenology](https://en.wikipedia.org/wiki/Phrenology), which are obviously racist, but we're also smart enough to realize that there are likely some ideas which we’ve internalized or been taught by a colonialist western society which are harmful to other minorities. We want to be able to learn about how everything we were taught was wrong, to be corrected, and that space can only exist when we don’t let people berate each other over ideas they project on others (whether that projection happens to be correct or not).
 
 ## Beehaw is not Reddit
-[Comments](https://beehaw.org/post/439918)
+[Comments](https://beehaw.org/post/439918https://beehaw.org/post/439918)
```

Comment:
I think this link change has a mistake in it

Reply:
Oh that was stupid. Thanks for catching.